test/DevHome.Test.csproj

```diff
@@ -17,6 +17,7 @@
     <PackageReference Include="MSTest.TestFramework" Version="2.2.10" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\database\DevHome.Database\DevHome.Database.csproj" />

```

Comment:
We don't want this under src?

Reply:
This is where I put the project.  It can move if someone wants this under src.